include/util/io.hpp

```diff
@@ -54,7 +54,8 @@ template <typename simple_type>
 bool serializeVectorIntoAdjacencyArray(const std::string &filename,
                                        const std::vector<std::vector<simple_type>> &data)
 {
-    std::ofstream out_stream(filename, std::ios::binary);
+	storage::io::FileWriter file(filename, storage::io::FileWriter::HasNoFingerprint);
```

Comment:
There's a clang-format script at scripts/format.sh - once your done with changes it would be great to run this.

(if you don't have clang-format we can do this for you before merging, too)